nx/include/switch/services/set.h

```diff
@@ -487,14 +487,38 @@ typedef struct {
     u8 padding[2];
 } SetSysDataBlock;
 
+typedef struct {
+    u8 extension_tag;                                        ///< 2 = CEA EDID timing extension, 0x70 = DisplayID Extension Block, 0xF0 = EDID Extension Block Map
+    union {
+        struct {
+            u8 revision;
+            u8 dtd_start;
+            u8 native_dtd_count : 4;
+            u8 native_dtd_feature_bitmap : 4;
+            SetSysDataBlock data_block;
+            SetSysModeLine extended_timing_descriptor[5];
+            u8 padding[5];
+        } NX_PACKED cea;                                     ///< CEA EDID timing extension
+        struct {
+            u8 data[0x7E];
+        } NX_PACKED display_id;                              ///< [13.0.0+] DisplayID Extension Block
```

Comment:
Shouldn't need these NX_PACKED with just u8s?

Reply:
You are right. Done.